app/main/lib/shared_models/paraphrase_multilingual_mpnet_base_v2.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from sentence_transformers import SentenceTransformer
+from flask import current_app as app
+
+from app.main.lib.shared_models.shared_model import SharedModel
+from app.main.lib.similarity_measures import angular_similarity
+
+class ParaphraseMultilingualMpnetBaseV2(SharedModel):
+    def load(self):
+        model_name = self.options.get('model_name', 'sentence-transformers/paraphrase-multilingual-mpnet-base-v2')
+        if self.options.get("model_url"):
+            try:
+                self.model = SentenceTransformer(self.options.get("model_url"))
+            except requests.exceptions.HTTPError as e:
+                app.logger.info('Attempting to load model by model name in lieu of broken URL')
+                self.model = SentenceTransformer(model_name)
```

Comment:
should we make this an __init__() function so it will initialize correctly on when class created?  Or is there a reason we defer until load() is called?

Reply:
I'm not sure why it was done this way, but I'm inclined to leave for now as all this code should be removed when Presto takes over